### ISSUE TITLE
Custom menus break cursor type in other places

### ISSUE THREAD

**Issue by joaomoreno** [Microsoft/vscode#53870]
When setting `"window.titleBarStyle": "custom"`, the mouse cursor is no longer `pointer` for the gear icon in the activity bar as well as status bar entries. It shows the default mouse cursor.

**Comment by eamodio** [comment]
I'm not seeing that behavior on Windows 10 with the latest insiders FYI.

**Comment by joaomoreno** [comment]
@eamodio Good point. Added the `linux` label, as I'm selfhosting on Linux.

**Comment by sbatten** [comment]
@joaomoreno I am not seeing this behavior on Linux in Stable nor latest insiders. Moving to August in case you are still seeing it.

**Comment by joaomoreno** [comment]
Yup, still see it. This is related to passing `frame: false` when creating a new `BrowserWindow`. Filed https://github.com/electron/electron/issues/13911

**Comment by joaomoreno** [comment]
Awesome, this seems to have been fixed recently: https://github.com/electron/electron/pull/13546

**Comment by joaomoreno** [comment]
@bpasero The issue was closed but not landed in 3, but 4. So, removing `fixed-in-electron-3.0.x`.

![image](https://user-images.githubusercontent.com/22350/49390546-8ca2cf00-f729-11e8-8310-fa6c88b9f27e.png)

**Comment by bpasero** [comment]
@joaomoreno are you sure you linked to the correct issue in Electron in https://github.com/Microsoft/vscode/issues/53870#issuecomment-409955901 ?

**Comment by joaomoreno** [comment]
@bpasero I created https://github.com/electron/electron/issues/13911, which was flagged as duplicate of https://github.com/electron/electron/issues/5723 which was said to be fixed by https://github.com/electron/electron/pull/13546 in https://github.com/electron/electron/issues/5723#issuecomment-408591913.

It's a big mess.

**Comment by joaomoreno** [comment]
@bpasero This is actually fixed!
